app/logic/courseManagement.py

```diff
@@ -77,5 +78,44 @@ def updateCourse(courseData):
     CourseInstructor.delete().where(CourseInstructor.course == course).execute()
     for instructor in instructorList:
         CourseInstructor.create(course=course, user=instructor)
+    createLog(f"Created SLC proposal: {courseData['courseName']}")
+    return course
+
+def updateCourse(courseData):
```

Comment:
Where did this function come from? It's suspicious if your merge conflict result is adding a whole new method, and not removing one from somewhere else. You have two `updateCourse` methods in the file now